pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -220,6 +220,11 @@ type ClusterSpec struct {
 	// +optional
 	// +nullable
 	LogCollector LogCollectorSpec `json:"logCollector,omitempty"`
+
+	// Image pull secrets to apply to all Ceph Pods
+	// +optional
+	// +nullable
+	ImagePullSecrets []v1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
```

Comment:
Similar to #10966, how about we move this setting under the `CephVersionSpec`? It is really related to the pulling of the image which is defined in that struct, and would follow the same pattern as adding the ImagePullPolicy in #10966.

Reply:
@ddl-kevin Planning on making this change so we can get this in? Thanks!

Reply:
Hey, sorry about this. We've gone in a different direction for our use case. I will see if I have some cycles this weekend to get this across the line.